README.md

```diff
@@ -472,6 +472,29 @@ Prints string to stdout, and returns string with additional utility methods
 like `.to()`.
 
 
+### mktemp([options,] [templates...])
+
+Available Options:
+
++ `-d`: Create a directory instead of a file.
++ `-u`: Unsafe mode. (Delete the file before retuning). Only use if you know what you're doing.
```

Comment:
I think it's best to just produce a random name. That's how the Linux command works, even if you can't put anything there. Otherwise, we run the risk of creating a file but not being able to delete it, or throwing an error when the file can't be created, when all it should do is make the name.

I would just mark it as the name "dry-run", since "unsafe" sounds like it means "insecure" (which really, it should be more secure since it shouldn't actually touch the filesystem). This is of course all based on the Linux behavior, and if Mac behavior actually checks if it's writable, let me know.